pkg/engine/resolve/resolve.go

```diff
@@ -734,7 +734,7 @@ func (r *Resolver) resolveArray(ctx *Context, array *Array, data []byte, arrayBu
 		r.byteSlicesPool.Put(arrayItems)
 	}()
 
-	_, err = jsonparser.ArrayEach(data, func(value []byte, dataType jsonparser.ValueType, offset int, err error) {
+	_, _ = jsonparser.ArrayEach(data, func(value []byte, dataType jsonparser.ValueType, offset int, err error) {
```

Comment:
I'm a bit confused by this line.

Reply:
As far as I understand there's no path which uses this returning error, so you can ignore it

Reply:
If the error path is never used, why does it even have the option to return an Error?

Reply:
it's not used in this case but of course can be used somewhere else. just specific to this usage